kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/CleanupPolicy.java

```diff
@@ -0,0 +1,29 @@
+package com.provectus.kafka.ui.model;
+
+import com.provectus.kafka.ui.exception.IllegalEntityStateException;
+import java.util.Arrays;
+
+public enum CleanupPolicy {
```

Comment:
You have these in contract, why do we need it?

Reply:
Here I added method for getting CleanupPolicy enum form string